presto-function-namespace-managers-common/src/main/java/com/facebook/presto/functionNamespace/AbstractSqlInvokedFunctionNamespaceManager.java

```diff
@@ -362,10 +363,13 @@ protected AggregationFunctionImplementation sqlInvokedFunctionToAggregationImple
                         "Need aggregationMetadata to get aggregation function implementation");
 
                 AggregationFunctionMetadata aggregationMetadata = function.getAggregationMetadata().get();
+                List<Type> parameters = function.getSignature().getArgumentTypes().stream().map(
```

Comment:
Remind me why we need this please?

Reply:
Its needed because we changed the `SqlInvokedAggregationFunctionImplementation` constructor here: https://github.com/prestodb/presto/blob/88e1152a75bf1d4edd555c197ab0c27e756d574e/presto-spi/src/main/java/com/facebook/presto/spi/function/SqlInvokedAggregationFunctionImplementation.java